backend/t4cclient/sessions/files/__init__.py

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
```

The license header got inserted twice (one is the old version and one is the new version).